src/flamenco/runtime/program/fd_bpf_loader_serialization.c

```diff
@@ -274,46 +287,65 @@ fd_bpf_loader_input_deserialize_aligned( fd_exec_instr_ctx_t ctx,
           return FD_EXECUTOR_INSTR_ERR_INVALID_REALLOC;
         }
 
-        /* https://github.com/anza-xyz/agave/blob/b5f5c3cdd3f9a5859c49ebc27221dc27e143d760/programs/bpf_loader/src/serialization.rs#L564-587 */
-        fd_borrowed_account_t * modify_acc = NULL;
-        int modify_err = fd_instr_borrowed_account_modify( &ctx, acc, post_len, &modify_acc );
-        if( FD_UNLIKELY( modify_err != FD_ACC_MGR_SUCCESS ) ) {
-          fd_valloc_free( ctx.valloc, buffer );
-          return FD_EXECUTOR_INSTR_ERR_ACC_BORROW_FAILED;
-        }
-
-        ulong pre_len = pre_lens[i];
-
-        start += fd_ulong_align_up( pre_len, FD_BPF_ALIGN_OF_U128 );
+        if( copy_account_data ) {
```

Comment:
nit: wonder if it would be clearer to use `direct_mapping_enabled` everywhere instead of `copy_account_data`.

Reply:
Was just matching up to agave logic